classes/class3/hw/views/catlistcolor.handlebars

```diff
@@ -0,0 +1,19 @@
+<center>
+	<h1><font size="7" color="#87CEFA">List of cats sorted based on their age and {{colorFilterMessage}} color</font></h1>
+</center>
+
+<ul>
+	{{#each cats}}
+	  <li>
+	  	<font size="4" color="#A52A2A">Name:</font> {{this.name}}<br>
+		<font size="4" color="#A52A2A">Age:</font> {{this.age}}<br>
+		<font size="4" color="#A52A2A">Colors:</font> {{this.colorlist}}<br><br>
```

Comment:
Oh no! Never use HTML styling! We haven't taught you CSS so we understand, but don't get used to this.